resources/styles/components/task/index.less

```diff
@@ -0,0 +1,35 @@
+.fixed-bar() {
+  left: 0;
+  position: fixed;
+  width: 100%;
+  background-color: fadeout(@tutor-white, 20%);
+}
+
+
+.task-view {
+  padding-top: 2 * @tutor-navbar-height;
+  padding-bottom: @tutor-navbar-height;
+
+  .panel-header {
+    .transition(transform 0.2s ease-out);
+
+    top: @tutor-navbar-height;
+    .fixed-bar();
+    padding: 1rem @tutor-navbar-padding-horizontal;
+    margin: 0;
+
+    button.task-details {
+      float: right;
+      position: static;
+    }
+  }
+  .panel-footer {
```

Comment:
Similarly, should this be a `.panel-footer` since:

1. it occurs inside `.panel-body`
2. is always pinned to the bottom